quaint/src/pooled.rs

```diff
@@ -390,12 +390,16 @@ impl Quaint {
             }
             #[cfg(feature = "mysql")]
             s if s.starts_with("mysql") => {
-                let url = crate::connector::MysqlUrl::new(url::Url::parse(s)?)?;
+                let mut url = crate::connector::MysqlUrl::new(url::Url::parse(s)?)?;
                 let connection_limit = url.connection_limit();
                 let pool_timeout = url.pool_timeout();
                 let max_connection_lifetime = url.max_connection_lifetime();
                 let max_idle_connection_lifetime = url.max_idle_connection_lifetime();
 
+                if is_tracing_enabled {
+                    url.query_params.statement_cache_size = 0;
+                }
```

Comment:
This needs to be documented.

Reply:
Same with Postgres. These three lines are a poor man's counterpart to https://github.com/prisma/prisma-engines/pull/5082.